MemoryManagement/src/memorymanagementAlgorithm/Segment.java

```diff
@@ -8,8 +17,18 @@
     private int base;
     private int limit;
     private boolean inserted = false;
-
 // ==============constructors ==========================//
+
+    //use this for gui holes

```

Comment:
will be never used. because it give an ID to hole and it's wrong.it must be removed or use a constructor parameter boolean isPartofProcess as I did.

Reply:
fixed